README.md

```diff
@@ -79,37 +61,63 @@ Or, you can add Lift to your `pom.xml` like so:
+
+We will accept issues and pull requests into the Lift codebase if the pull requests meet the following criteria:
+
+* The request is for a [supported version of Lift][supfile]
+* The request adheres to our [contributing guidelines][contribfile], including having been discussed
+  on the Mailing List if its from a non-committer.
+
+[contribfile]: https://github.com/lift/framework/blob/master/CONTRIBUTING.md
+[sigfile]: https://github.com/lift/framework/blob/master/contributors.md
+
+## Project Organization
 
-#### web
+The Lift Framework is divided into several components that are published independently. This organization enables you to use just the elements of Lift necessary for your project and no more.
 
-This component includes all of Lift's core HTTP and web handling. Including `lift-webkit` in your build process should be sufficient for basic applications and will include `lift-core` as a transitive dependency.
+### This Repository
 
-#### persistence
+This repository, `framework`, contains the following components:
 
-This component includes Mapper and Record, Lift's two ORMs. While you needn't use either and can use the ORM of your choice, Mapper and Record integrate nicely with Lift's idioms. Mapper is an ORM for relational databases, while Record is a broader ORM with support for both SQL databases and NoSQL datastores.
+* **core:** Core elements used by Lift projects. If you wish to reuse some of Lift's helpers and
+constructs, such as `Box`, this component may be all you need. However, a web application will most
+likely require one or more of Lift's components.
+* **web:** This component includes all of Lift's core HTTP and web handling. Including `lift-webkit`
+in your build process should be sufficient for basic applications and will include `lift-core` as a
+transitive dependency.
+* **persistence:** This component includes Mapper and Record, Lift's two ORMs. While you needn't use
+either and can use the ORM of your choice, Mapper and Record integrate nicely with Lift's idioms.
+Mapper is an ORM for relational databases, while Record is a broader ORM with support for both SQL
+databases and NoSQL datastores.
 
 ### Other Repostories
 
 There are a variety of other repositories available on the Lift GitHub page. While many are concerned with building Lift or are build program archetypes, there are two you will probably encounter fairly frequently as a Lift user:
 
 #### modules
 
-The [modules](https://github.com/liftmodules) repository contains the many add-on modules that are part of the Lift project. If you don't find a module you need here, consider [creating a module](http://www.assembla.com/spaces/liftweb/wiki/Modules) and sharing it with the community.
-
-Please note that the modules project does accept pull requests.
+The [modules](https://github.com/liftmodules) organization contains the many add-on modules that are
+part of the Lift project. If you don't find a module you need here, consider
```

Comment:
... If you don't find a module you need here see also [https://liftweb.net/lift_modules](https://liftweb.net/lift_modules) or consider ...